packages/query/src/graphql/plugins/PgAggregationPlugin.ts

```diff
@@ -0,0 +1,107 @@
+// Copyright 2020-2021 OnFinality Limited authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import AddAggregateTypesPlugin from '@graphile/pg-aggregates/dist/AddAggregateTypesPlugin';
+import AddConnectionAggregatesPlugin from '@graphile/pg-aggregates/dist/AddConnectionAggregatesPlugin';
+import AddConnectionGroupedAggregatesPlugin from '@graphile/pg-aggregates/dist/AddConnectionGroupedAggregatesPlugin';
+import AddGroupByAggregateEnumsPlugin from '@graphile/pg-aggregates/dist/AddGroupByAggregateEnumsPlugin';
+import AddGroupByAggregateEnumValuesForColumnsPlugin from '@graphile/pg-aggregates/dist/AddGroupByAggregateEnumValuesForColumnsPlugin';
+import AddHavingAggregateTypesPlugin from '@graphile/pg-aggregates/dist/AddHavingAggregateTypesPlugin';
+import AggregateSpecsPlugin from '@graphile/pg-aggregates/dist/AggregateSpecsPlugin';
+import FilterRelationalAggregatesPlugin from '@graphile/pg-aggregates/dist/FilterRelationalAggregatesPlugin';
+import InflectionPlugin from '@graphile/pg-aggregates/dist/InflectionPlugin';
+import OrderByAggregatesPlugin from '@graphile/pg-aggregates/dist/OrderByAggregatesPlugin';
+
+import type {Plugin} from 'graphile-build';
+import type {PgAttribute, PgType, SQL, PgProc} from 'graphile-build-pg';
+import {makePluginByCombiningPlugins} from 'graphile-utils';
+
+import {argv} from '../../yargs';
+
+const unsafe = argv('unsafe') as boolean;
+
+interface AggregateGroupBySpec {
```

Comment:
you don't need to redefine the interfaces

Reply:
Yeah I missed that they were exported.